tests/jointcalTestBase.py

```diff
@@ -0,0 +1,311 @@
+# See COPYRIGHT file at the top of the source tree.
+
+from __future__ import division, absolute_import, print_function
+
+import numpy as np
+import collections
+import os
+
+from lsst.meas.astrom import LoadAstrometryNetObjectsTask, LoadAstrometryNetObjectsConfig
+from lsst.afw import geom, table
+
+from lsst.jointcal import jointcal
+
+arcsec_per_radian = 206265.
+
+
+class JointcalTestBase(object):
```

Comment:
Just what @timj said here. If the class doesn't have any tests, it shouldn't derive from TestCase.